pathod/pathod.py

```diff
@@ -71,7 +69,7 @@ def get_cert(self, name):
 
 class PathodHandler(tcp.BaseHandler):
     wbufsize = 0
-    sni = None
+    sni = None  # type: typing.Union[str, None, bool]
```

Comment:
> sni: ``str`` for a custom SNI value,
                ``None`` for the client SNI value,
                ``False`` if no SNI value should be sent.